### DISCUSSION TITLE
micro-nightly hash check failed

### DISCUSSION THREAD

**Discussion by bachig26** [ScoopInstaller/Versions#564] [General]
This issue always happens with the micro nightly on every new update.

It's very difficult to open a issue every time to fix this.

Is there a workaround or fix to the versions micro-nightly package to solve this permanently?

**Comment by tech189** [comment]
That's annoying, they seem to be keeping the same file name but uploading a new build. I'll look at ways to add the current date to the version number to keep the hashes up-to-date.

**Reply by tech189** [reply]
@bachig26 I've made a fix in https://github.com/ScoopInstaller/Versions/commit/2f17c8c9b3c8e3df43cbecef834ef2dca144065a. It should update to the latest whenever there's a new file uploaded, even with the same version number. Please be aware that our GitHub autoupdate action runs every 4 hours so there will still be some delay. Try it over the next few days and let me know if it's working!

**Reply by bachig26** [reply]
@tech189 I've tried with this change and the update is working properly. No problem until now. 
Thank you. 👍

**Reply by tech189** [reply]
@bachig26 Glad to hear it, thank you for letting me know!

**Reply by tfer** [reply]
I keep running into a hash check error, was not aware that this might be the source, i.e. that the hash may not have updated yet due to GitHub autoupdate lagging the file update.

- Perhaps change:

	* Please try again or create a new issue by using the following link and paste your console output: ...
- to:
	* Please try again later as hashes recomputed every 4 hours and may be lagging, if it persists create a new issue by using the following link and paste your console output:

**Reply by tfer** [reply]
Not sure that this is in effect anymore, all of my submitted issues for this, (5), seem to have been "hand" fixed though the contributor that closes them is the GitHub actions bot.

**Comment by bachig26** [comment]
@tech189 #645 this issue came back again.
